kayenta-stackdriver/src/main/java/com/netflix/kayenta/stackdriver/canary/StackdriverCanaryScopeFactory.java

```diff
@@ -49,6 +49,8 @@ public CanaryScope buildCanaryScope(CanaryScope canaryScope){
       }
 
       stackdriverCanaryScope.setResourceType(extendedScopeParams.getOrDefault("resourceType", "gce_instance"));
+      stackdriverCanaryScope.setCrossSeriesReducer(extendedScopeParams.getOrDefault("crossSeriesReducer", "REDUCE_MEAN"));
+      stackdriverCanaryScope.setPerSeriesAligner(extendedScopeParams.getOrDefault("perSeriesAligner", "ALIGN_MEAN"));
```

Comment:
Why not make the default the current stackdriver scope's value?

Reply:
Changed it to only set the values if they exist in `extendedScopeParams`.